Assets/Scripts/UI/LogText/Injury.cs

```diff
@@ -36,11 +38,54 @@ public override IEnumerable<LogRecord> Convert(Ffb.Dto.Reports.Injury report)
             if (report.armorRoll?.Length > 0)
             {
                 yield return new LogRecord($"<b>Armour Roll [ {report.armorRoll[0]} ][ {report.armorRoll[1]} ]</b>");
+                yield return new LogRecord($"<b>Rolled Total of {report.armorRoll[0] + report.armorRoll[1]}</b>");
 
-                //
-                // TODO: Incomplete
-                //
-            }
+                int armorModifierTotal = 0;
+                int i = 0;
+
+                var status = "";
+                int rolledTotal = report.armorRoll[0] + report.armorRoll[1];
+
+                foreach (FFBEnumeration entry in report.armorModifiers)
+                {
+                   var armourModifier = entry.AsArmorModifier();
```

Comment:
With the new structure of FFBEnumerations, this should read:

var armourModifier = entry.As<ArmorModifier>();

Reply:
Sounds good. I think I must have missed the updates to entry.As() by an hour or so. I will update.